dodo.py

```diff
@@ -9,10 +9,10 @@ def integration_test_action():
         # printing to stderr so that doit does not capture the output
         print("### Integration tests are not yet implemented ###", file=sys.stderr)
 
-    for kind, action in [('unit', unittests.run_all_unit_tests), ('integration', integration_test_action)]:
+    for kind, action in [('unit', ['coverage run unittests.py']), ('integration', [integration_test_action])]:
```

Comment:
Just to understand what is happening: is the `unittests.run_all_unit_tests` still called? Do we still get report on the testrun itself? What happens if a test fails? Does `coverage` report this, or does it only give a reduced coverage?

Reply:
I'm curious: could you break the code somewhere, commit it, so we can see the output

Reply:
I think having at least one broken commit in the history would be good practice for PRs that concern testing, right?

Reply:
I had to rebase this branch, since the branch hat conflicts to `master` produced by the merge of #25 .

I added a broken commit bc3e4f2a05c97bf1b4bbafe9a6cf19252d219b52 (contains a fix too, but makes the build break) and a fix commit 99db918d126c83dafc0bb8bca5f5886d807684a2. You should find them in the history printed on this page. The first commit shows that the Travis CI build fails.